.github/workflows/supabase-migrate.yml

```
@@ -0,0 +1,232 @@
+# user/account, not the project; the project ref + DB password choose which
+# project the push targets. Per-environment scoping of refs/passwords is what
+# enforces the tier boundary.
+
+on:
+  push:
+    branches: [main, staging]
+    paths:
+      - "products/dashboard/supabase/**"
+      - ".github/workflows/supabase-migrate.yml"
+  pull_request:
+    paths-ignore: []
```

actionlint (1.7.12)

[error] "paths-ignore" section should not be empty

(syntax-check)